lib/hardware_services/inventory_service.py

```diff
@@ -32,9 +32,19 @@
 # will be able to continue with minimal changes (only thing that this will change is where QUADS pulls its data from)
 #####################################################################################################################
 
-import sys
+import os, sys, importlib
 from abc import ABCMeta, abstractmethod
 
+
+sys.path.append(os.path.dirname(__file__) + "/hardware_services/inventory_drivers/")
+
+
+def get_inventory_serivce(name, url):
+    inventoryservice = name + "InventoryDriver"
+    importlib.import_module()
+    return getattr(sys.modules[inventoryservice], inventoryservice)(url)
```

Comment:
Is it fine to have "sys.path.append()" in many places in code?

Can we do this instead?

def my_import(name):
    components = name.split('.')
    mod = __import__(components[0])
    for comp in components[1:]:
        mod = getattr(mod, comp)
    return mod